install.sh

```diff
@@ -647,7 +647,7 @@ killtree() {
 }
 
 getshims() {
-    ps -e -o pid= -o args= | sed -e 's/^ *//; s/\s\s*/\t/;' | grep -w 'k3s/data/[^/]*/bin/containerd-shim' | cut -f1
```

Comment:
I think the concern with not being more specific was that we could kill shims managed by other things - for example, the containers started by a host-level install of containerd+docker. We only want to kill things we started. Can you confirm that your change doesn't alter this behavior? If you run this modified command on a system with docker, do all the docker containers continue to run?